tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -219,6 +220,17 @@ policy_refresh_cagg_proc(PG_FUNCTION_ARGS)
 	PG_RETURN_VOID();
 }
 
+Datum
+policy_refresh_cagg_check(PG_FUNCTION_ARGS)
+{
+	if (PG_NARGS() != 1 || PG_ARGISNULL(0))
+		PG_RETURN_VOID();
```

Comment:
The return line is not covered. Might be good to have an actual test that this function does generate an error and does not cause a crash or something else.